pom.xml

```diff
@@ -53,6 +44,7 @@
 							<outputDirectory>${basedir}/target/OpenHospital20/rsc</outputDirectory>
 							<resources>
 								<resource>
+									<directory>src/main/resources</directory>
```

Comment:
I think this is the [standard location](http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html) and therefore can be omitted (but I might be wrong, haven't tested it).

Reply:
this is in the `<goal>copy-resources</goal>` so it is needed to build the target I think and not for Maven itself